common/core/src/commonMain/kotlin/com/trafi/core/model/FilterItem.kt

```diff
@@ -0,0 +1,24 @@
+package com.trafi.core.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ *
+ * @param id
+ * @param name
+ * @param icon
+ * @param color
+ * @param accentColor
+ * @param transportsIds
+ */
+@Serializable
+data class FilterItem(
```

Comment:
Have these been added to the platform-api contract? If no, we'll lose this file when we regenerate the models 😟